includes/Assets.php

```diff
@@ -94,7 +94,9 @@ public function enqueue_admin_scripts( $hook ) {
             }
 
             // load fontawesome styles
-            wp_enqueue_style( 'dokan-fontawesome' );
+            if ( 'off' === dokan_get_option( 'disable_dokan_fontawesome', 'dokan_appearance', 'off' ) ) {
```

Comment:
kindly revert the previous code, since we are not removing fontawesome from dokan admin page